### PR TITLE
Symantec DLP Integration

### PR DESCRIPTION
**Pull request by guyfreund**
## Status
In Progress

## Related Issues
fixes: https://github.com/demisto/etc/issues/4911

## Description
Symantec DLP Integration

## Must have
- [x] Tests
- [x] Documentation (with link to it)
- [x] Code Review

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](https://github.com/demisto/content/blob/10aa66b84b777bdfb79ffef5d24d7801bb7afc76/Beta_Integrations/SymantecDLP/SymantecDLP.yml)
- [ ] [README file](https://github.com/demisto/content/blob/10aa66b84b777bdfb79ffef5d24d7801bb7afc76/Beta_Integrations/SymantecDLP/README.md)

### CODE REVIEW

File: Integrations/SymantecDLP/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## [Unreleased]
+#### New Integration
+Cloud-based SaaS to discover, monitor, and protect sensitive data.
```
Comment (Itay4) [inline Integrations/SymantecDLP/CHANGELOG.md]: no need for changelog in new integration

File: TestPlaybooks/playbook-Symantec_Data_Loss_Prevention_Test.yml [hunk 2]
```diff
@@ -0,0 +1,487 @@
+id: Symantec Data Loss Prevention-Test
```
Comment (Itay4) [inline TestPlaybooks/playbook-Symantec_Data_Loss_Prevention_Test.yml]:
```suggestion
id: Symantec Data Loss Prevention - Test
```

File: TestPlaybooks/playbook-Symantec_Data_Loss_Prevention_Test.yml [hunk 3]
```diff
@@ -0,0 +1,487 @@
...
+version: -1
+name: Symantec Data Loss Prevention-Test
```
Comment (Itay4) [inline TestPlaybooks/playbook-Symantec_Data_Loss_Prevention_Test.yml]:
```suggestion
name: Symantec Data Loss Prevention - Test
```

File: Integrations/SymantecDLP/SymantecDLP.yml [hunk 4]
```diff
@@ -0,0 +1,453 @@
+category: Endpoint
```
Comment (Itay4) [inline Integrations/SymantecDLP/SymantecDLP.yml]: update category

Comment (Itay4) [inline Integrations/SymantecDLP/SymantecDLP.yml]: add fromversion: 4.5.0

**Comment by guyfreund** [comment]
@kirbles19 - The YML & README files are ready to be reviewed.

**Comment by Itay4** [comment]
@richardbluestone please review the yml

File: Beta_Integrations/SymantecDLP/SymantecDLP.yml [hunk 5]
```diff
@@ -0,0 +1,460 @@
...
+  id: Symantec Data Loss Prevention
+  version: -1
+configuration:
+- display: Enforce Server (e.g. https://192.168.0.1)
+  name: server
+  required: true
+  type: 0
+- display: Username
+  name: credentials
+  required: false
```
Comment (Itay4) [inline Beta_Integrations/SymantecDLP/SymantecDLP.yml]:
```suggestion
  required: true
```

File: Beta_Integrations/SymantecDLP/SymantecDLP.yml [hunk 6]
```diff
@@ -0,0 +1,460 @@
...
+    - contextPath: SymantecDLP.Incident.ViolatingComponent.ViolatingSegment.Text.Type
+      description: The type of data that triggered the violation.
+      type: String
+    - contextPath: SymantecDLP.Incident.ViolatingComponent.ViolatingSegment.Text.RuleID
+      description: The rule ID which triggered the violation.
+      type: Number
+    - contextPath: SymantecDLP.Incident.ViolatingComponent.ViolatingSegment.Text.RuleName
+      description: The rule name which triggered the violation.
+      type: String
+  dockerimage: demisto/zeep:1.0.0.1281
```
Comment (Itay4) [inline Beta_Integrations/SymantecDLP/SymantecDLP.yml]:
```suggestion
  dockerimage: demisto/zeep:1.0.0.3833
```

File: Beta_Integrations/SymantecDLP/SymantecDLP.py [hunk 7]
```diff
@@ -0,0 +1,725 @@
...
+        'symantec-dlp-list-custom-attributes': list_custom_attributes_command,
+        'symantec-dlp-list-incident-status': list_incident_status_command,
+        'symantec-dlp-incident-violations': incident_violations_command
+    }
+    try:
+        handle_proxy()
+        if command == 'fetch-incidents':
+            fetch_incidents(client, fetch_time, fetch_limit, last_run, saved_report_id)  # type: ignore
+        elif command == 'test-module':
+            test_module(client, saved_report_id)  # type: ignore
```
Comment (Itay4) [inline Beta_Integrations/SymantecDLP/SymantecDLP.py]: add error code https://github.com/demisto/content/tree/master/docs/linting#mypy

File: Beta_Integrations/SymantecDLP/SymantecDLP.py [hunk 8]
```diff
@@ -0,0 +1,725 @@
...
+    fetch_time: str = params.get('fetch_time', '3 days').strip()
+    try:
+        fetch_limit: int = int(params.get('fetch_limit', '10'))
+    except ValueError:
+        raise DemistoException('Value for fetch_limit must be an integer.')
+    saved_report_id: str = demisto.params().get('saved_report_id', '')
+    last_run: dict = demisto.getLastRun()
+    args: dict = demisto.args()
+    verify_ssl = not params.get('insecure', False)
+    # proxy = params.get('proxy')
```
Comment (Itay4) [inline Beta_Integrations/SymantecDLP/SymantecDLP.py]: ?

**Comment by richardbluestone** [comment]
I've reviewed the yml file.

**Pull request merged**